src/ex3/todoApp.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
```

Comment:
well done that this cli handling file is clean of logic and only call business logic methods